src/createContext.tsx

```diff
@@ -48,7 +48,10 @@ function createProvider<State>(
     const state = React.useState(initialState);
     const value = React.useMemo(() => state, [state[0]]);
 
-    useDevtools(state, { enabled: devtools });
+    useDevtools(state, {
+      enabled: devtools != null,
```

Comment:
This won't work if we do `<Provider devtools={false} />`.
```suggestion
      enabled: Boolean(devtools),
```